apps/utils/locust_tests/requirements.txt

```diff
@@ -781,9 +781,9 @@ typing-extensions==4.8.0 \
     --hash=sha256:8f92fc8806f9a6b641eaa5318da32b44d401efaac0f6678c9bc448ba3605faa0 \
     --hash=sha256:df8e4339e9cb77357558cbdbceca33c303714cf861d1eef15e1070055ae8b7ef
     # via locust
-urllib3==1.26.16 \
-    --hash=sha256:8d36afa7616d8ab714608411b4a3b13e58f463aee519024578e062e141dce20f \
-    --hash=sha256:8f135f6502756bde6b2a9b28989df5fbe87c9970cecaa69041edcce7f0589b14
+urllib3==1.26.17 \
+    --hash=sha256:24d6a242c28d29af46c3fae832c36db3bbebcc533dd1bb549172cd739c82df21 \
+    --hash=sha256:94a757d178c9be92ef5539b8840d48dc9cf1b2709c9d6b588232a055c524458b
```

Comment:
It looks like this _does_ work, but if we were to ever update package version pins in the corresponding `requirements.in` without _also_ pinning `urllib3` to this new version in `requirements.in`, we may end up reverting to an older version of `urllib3` when that `requirements.in` is `pip-compile`'d.

We can pair on doing this upgrade using `pip-compile` so that we don't end up accidentally downgrading this `urllib3` version in the future.

Reply:
> It looks like this _does_ work, but [...]

Thanks for taking a closer look, @malessi.

Based on what's here, can we assume that @aadedejifearless just neglected to check-in changes to `requirements.in`. I'm not clear on how they achieved this result unless... maybe the hashes were lifted from one of the defunct, dependabot change sets? 🤔 

I recall that we had gone to the trouble of making these GHA workflows more meaningful, but those tests only evaluate a subset of the desired state, i.e. partial functionality, **not** maintainability. I'll be more careful before lending my approval on these in the future. I'm curious if a `pip-compile` step to ensure the change set is diff-free could be wise. Then again, maybe that's too much...

Reply:
This is my bad! - Ade reached out and I told them to use the `hashin` library to generate the hashes. I didn't know/remember about `pip-compile`. TIL :)

Reply:
Added a page in the wiki so we can always reference if changes are needed.

Reply:
https://github.com/CMSgov/beneficiary-fhir-data/wiki/How-to-Perform-Manual-Updates-to-Locust-Test-Dependencies